modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/SmallImage.js

```diff
@@ -120,9 +120,9 @@ export default function SmallImage({
 							/>
 
 							<ClayInput
-								onClick={() => fileInputRef.current?.click()}
```

Comment:
Why did we remove the onClick here?

Reply:
Hi @bryceosterhaus! In terms of accessibility, inputs shouldn't open modals, since it's not the correct trigger to open a modal. Even if this were necessary, we should add the correct roles and arias so that the user understands that this opens a modal. However, in these cases we have an additional button that already opens the item selector modal, so it would not be necessary to do it in the input as well. 
This has already been done previously on other sites on the portal https://liferay.atlassian.net/browse/LPS-198726

Reply:
Great, good to know. Thanks!